test-util/util.js

```diff
@@ -216,10 +216,10 @@ function testRunsAndHasCheckBlocks(it, name, toEval, specs) {
       self.browser.wait(function () {
         return self.browser.isElementPresent(webdriver.By.className("check-results-done-rendering"));
       }, 20000);
-      return response.findElements(webdriver.By.className("check-block-result"));
+      return response.findElements(webdriver.By.className("check-block"));
     });
     var blocksAsSpec = checkBlocks.then(function(cbs) {
-      var tests = cbs.map(function(cb, i) {
+      var tests = cbs.slice(1).map(function(cb, i) {
```

Comment:
I moved the summary to the top of the repl (since results are added eagerly, from top-to-bottom, roughly) and its class name is `check-block`.